packages/amplify_core/ios/Classes/SwiftCore.swift

```diff
@@ -49,15 +52,19 @@ public class SwiftCore: NSObject, FlutterPlugin {
   }
 
     private func onConfigure(result: FlutterResult, version: String, amplifyConfiguration: AmplifyConfiguration) {
-    do {
-        
-      AmplifyAWSServiceConfiguration.addUserAgentPlatform(.flutter, version: version)
-      try Amplify.configure(amplifyConfiguration)
-      result(true)
-    } catch {
-      result(FlutterError(code: "AmplifyException",
-                          message: "Failed to Configure Amplify",
-                          details: error.localizedDescription));
-    }
+      if (!isConfigured) {
```

Comment:
Would it be useful for the native platforms to add `Amplify.isConfigured()`? (Related: https://github.com/aws-amplify/amplify-flutter/issues/124)

Reply:
@jamesonwilliams I think we can handle this on the flutter side without too much bloat; I can't really speak to whether this change adds value to the amplify-android library itself.